File: IOSwiftUI/Common/Models/OneTimeCodes/BOOneTimeCodeModel.cs
using System;
using System.ComponentModel.DataAnnotations;
using IOSwiftUI.Common.Enumerations;
using IOSwiftUI.Common.Models.Base;

namespace IOSwiftUI.Common;

public class BOOneTimeCodeModel : Model
{
    public int ID { get; set; }

    [Required]
    [StringLength(16)]
    public string PhoneNumber { get; set; }

    [Required]
    [StringLength(6)]
    public string OneTimeCode { get; set; }

    [Required]
    public DateTimeOffset CreateDate { get; set; }

    public DateTimeOffset? ValidateDate { get; set; }


}

File: IOSwiftUI/Backoffice.UI/src/screens/onetimecodes/models/BOOneTimeCodeModel.ts


class BOOneTimeCodeModel {

    id: number | null;
    phoneNumber: string;
    oneTimeCode: string;
    createDate: string;
    validateDate: string | null;

    constructor() {
        this.id = null;
        this.phoneNumber = "";
        this.oneTimeCode = "";
        this.createDate = "";
        this.validateDate = null;

    }
}

export default BOOneTimeCodeModel;
